### PR TITLE
some minor cppcheck fixes

### PR DESCRIPTION
**Pull request by cujomalainey**

### CODE REVIEW

**Review by ranj063 — APPROVED** [review]

**Review by RanderWang — APPROVED** [review]

**Review by marcinszkudlinski — APPROVED** [review]

**Review by mwasko — CHANGES_REQUESTED** [review]
@cujomalainey can you fix a spelling "uninitialized" to satisfy check-patch?

**Comment by cujomalainey** [comment]
last checkpatch error is the scripts fault

**Review by lgirdwood — APPROVED** [review]
@cujomalainey is there an equivalent PR for the kernel IPC header ? If not maybe @ujfalusi can quickly submit one to align the ABI.

**Comment by cujomalainey** [comment]
ah no, I was working on this late and forgot. I also think the kernel might not be hit by the same bug as they are typically 64bit, bit regardless adding the change wouldn't hurt it.

**Review by mwasko — APPROVED** [review]

**Comment by lgirdwood** [comment]
ok, everything looks good now, lets wait for a kernel PR and merge once both approved.

File: src/ipc/ipc4/handler.c
Comment (lyakh) [inline src/ipc/ipc4/handler.c:372]: personally, I'd just use `return 0;` at line 402 and even moved the `ret` variable into the scope where it's used - inside the loop

Comment (cujomalainey) [reply src/ipc/ipc4/handler.c:372]: that would require modifying the logic at 399 as well since it drops its return value down.

**Comment by cujomalainey** [comment]
Kernel PR posted, see reference above this comment

**Pull request merged**
